transcendence-app/src/chat/dto/update-chat.dto.ts

```diff
@@ -0,0 +1,18 @@
+import { Transform, TransformFnParams } from 'class-transformer';
+import { IsInt, IsNotEmpty, IsOptional, IsString } from 'class-validator';
+
+export class UpdateChatDto {
+  @IsString()
+  @IsNotEmpty()
+  @Transform(({ value }: TransformFnParams) => value?.trim())
+  @IsOptional()
+  name?: string;
+
+  @IsInt()
+  @IsOptional()
+  avatarX?: number;
+
+  @IsInt()
+  @IsOptional()
+  avatarY?: number;
+}
```

Comment:
I think we may rethink a bit how we implement avatars. They are also going to be used by chat and it may make sense to create a module for this in the future